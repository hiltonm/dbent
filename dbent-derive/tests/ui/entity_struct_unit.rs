use dbent::prelude::*;

#[derive(Entity)]
struct Test();

fn main() {}
