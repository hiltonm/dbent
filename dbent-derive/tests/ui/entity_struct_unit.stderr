error: #[derive(Entity)] can only be used on structs with named fields
 --> tests/ui/entity_struct_unit.rs:4:12
  |
4 | struct Test();
  |            ^^
